alacritty/src/display/mod.rs

```diff
@@ -1413,7 +1469,7 @@ impl Drop for Display {
     fn drop(&mut self) {
         // Switch OpenGL context before dropping, otherwise objects (like programs) from other
         // contexts might be deleted when dropping renderer.
-        self.make_current();
+        self.make_current(false);
```

Comment:
Are we just not recovering here because it's not necessary? Seems kinda odd to have a parameter solely for the drop impl.

Reply:
yeah, we don't recover since it doesn't matter, otherwise we create render and drop it.

Reply:
Would *probably* be fine tbh. It's **extremely** unlikely to ever happen anyway.

Like who manages to shut down Alacritty right between rendering at a billion frames a second and a GPU reset? Even if we crash at that point most people would never notice.